src/common/settings.ts

```diff
@@ -104,14 +106,20 @@ export async function getWorkspaceSettings(
 export function checkIfConfigurationChanged(e: ConfigurationChangeEvent, namespace: string): boolean {
     const settings = [
         `${namespace}.check`,
-        `${namespace}.trace`,
```

Comment:
Is this a permanent change? Just curious about why the switch away from their set log level.

Reply:
this was a missed change when we switched to logLevel setting. We moved from `trace` to `logLevel` because language server itself internally uses `trace` setting. Our setting was overriding that. We are eventually going to get rid of all logging, and replace it with VS Codes own logging.

Reply:
thank you for the explanation sounds good!